Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx

```diff
@@ -161,6 +161,21 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
     }
   }
 
+  private sendingNumbers = (message: any) => {
+    let currentNum = message.succeeded + message.failed + message.throttled;
```

Comment:
Move this line into the "else" block just above the return.

Reply:
done